src/script/cbottoken.cpp

```diff
@@ -99,6 +99,10 @@ const char* GetObjectName(ObjectType type)
     if ( type == OBJECT_MOBILEts    )  return "TrackedSniffer";
     if ( type == OBJECT_MOBILEfs    )  return "WingedSniffer";
     if ( type == OBJECT_MOBILEis    )  return "LeggedSniffer";
+    if ( type == OBJECT_MOBILEwb    )  return "WheeledBuilder";
```

Comment:
Oh, and this PR doesn't even compile :stuck_out_tongue:

Reply:
Now it compiles.